graylog2-web-interface/src/views/bindings.test.jsx

```diff
@@ -155,5 +156,51 @@ describe('Views bindings', () => {
           .toEqual(false);
       });
     });
+    describe('CreateExtractor', () => {
```

Comment:
This is now part of the `field actions` `describe`-block, although it is a value action. I think this is a good time to either introduce a new `value actions` block, or split the file into two, `bindings.fieldActions.test.jsx` & `bindings.valueActions.test.jsx`.

Reply:
Good catch! Because we are planing to split up the bindings in the future, I've create two separate test files. When we are adding even more separate test files, it would be okay, to move them inside a `__tests__` dir, right?

Reply:
Sure!